packages/trpc/server/routers/viewer/bookings/requestReschedule.handler.ts

```diff
@@ -249,7 +253,7 @@ export const requestRescheduleHandler = async ({ ctx, input }: RequestReschedule
 
   log.debug("builder.calendarEvent", safeStringify(builder.calendarEvent));
   // Send emails
-  await sendRequestRescheduleEmail(builder.calendarEvent, {
+  await sendRequestRescheduleEmailAndSMS(builder.calendarEvent, {
```

Comment:
`team` is not defined on `builder.calendarEvent` so SMS is not sending

Can you check that for all other sendEmailAndSMS functions as well?